helm/stargate/templates/stargate_graphql_deployment.yaml

```diff
@@ -0,0 +1,51 @@
+{{ if .Values.graphqlapi.enabled }}
```

Comment:
same comments as for the docsapi

Reply:
Override of port number not needed to be in sync with stargate documentation.